README.md

```diff
@@ -8,14 +8,20 @@ The MDIS wizard contains all CPUs and carrier boards for configuring a system.
 Building MDIS Wizard:
 ---------------------
 
-to build the MDIS Wizard the repo must be checked out on the old software server (192.1.1.22) because
-of QT and tool chain available only there.
+The MDIS Wizzard`s code is based on fairly old QT4 code with QT3 support in it. To build it a compatible
+compilation sysroot is necessary. This sysroot comes from an old software server at MeN (used to
+be at 192.1.1.22) and it contains a Qt 4.6.0 compilation in it and all the static libraries that are
+needed to create the binary (X11, fontconfig, etc.).
+
+The location of these sysroot needs to be defined in the `builddist.env` file. The variable `QTSYSROOT`
+tells the Makefile where to find all the necessary dependencies and tools.
```

Comment:
Additionally, we should tell in the README that the package gcc-multilib should be installed for 32bit support

Reply:
We should not need any package and everything should come from the sysroot. @mad-jrodriguez  and myself have found and issue where actually it was a missing -I option for the compiler so everything is found in the compilation sysroot.

Reply:
Yes, you are right. Now, it works properly, only adding -I$(QTSYSROOT)/usr/include